vuepress/docs/next/tutorials/samples/customer-portal.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebarDepth: 2
+---
+
+# Entando Customer Portal 
+## Overview
+The Entando Customer Portal enables an organization to quickly provide a modern, self-service customer-facing application for management of subscriptions. It includes a lightweight integration to JIRA Service Management for access to service tickets and a Role Based Access Control (RBAC) design for granting users varying levels of access.
+
+Key Features:
+
+* Customizable service ticket and tracking system with Jira Service Management
+* Role-based access control (RBAC) with Entando Identity Management System
+* Integrated user, customer, project, and subscription management 
+
+This tutorial covers: 
+1. [Installation](#installation)
+2. [Configuration](#configuration)
+4. [Managing the Customer Portal](#managing-the-customer-portal)
+5. [Using the Customer Portal](#using-the-customer-portal)
+6. [Application Details](#application-details)
+7. [Resources](#resources)
+
+## Installation
+### Prerequisites
+* A working instance of Entando running on Kubernetes. See [Getting Started](../../docs/getting-started/) for more information or [install Entando on any Kubernetes provider](../#operations). 
+* Use the Entando CLI command `ent check-env develop` to verify all dependencies
+* Docker installed and running
```

Comment:
Docker isn't needed unless someone is building the Customer Portal so this can be removed.

Reply:
should this be noted in the manual installation section?

Reply:
@jyunmitch Nope, it's only required if you're going to build your own copy of the Customer Portal from source. We have some tips on that in the PDA doc but I don't think it's needed here. That's the kind of thing I would definitely point them to the source repository on rather than try to document it here.